ome/omero-iviewer
# Plane roi grid

This allows you to browse ROIs by plane (find which planes have most ROIs on them).
It implements most of what's on https://github.com/openmicroscopy/design/issues/91

To test:
 - Find a multi-plane image and make sure you have various numbers of Shapes on different planes, including some shapes 'unattached' to Z or T.
 - Under the ROIs tab, click on the "Planes" sub-tab.
 - The grid should show numbers of shapes that appear on each Z/T plane.
 - The 'Key' should show the range of Shape counts per plane
 - Tool-tip on each grid 'cell' should give you the count for that plane.
 - Clicking on the plane should take you to that Z/T position (and show the cell selected)
 - Shift-clicking on a Z-stack image should select a Z-projection range.
 - Single-click (no Shift) should turn off the Z-projection again.

Demo at https://twitter.com/will_j_moore/status/1103206476695236609
